scalding-core/src/main/scala/com/twitter/package.scala

```diff
@@ -31,6 +32,8 @@ package object scalding {
   type ValuePipe[+T] = com.twitter.scalding.typed.ValuePipe[T]
   type Grouped[K, +V] = com.twitter.scalding.typed.Grouped[K, V]
 
+  implicit def typedDescriptor[T]: TypeDescriptor[T] = macro com.twitter.scalding.macros.impl.TypeDescriptorProviderImpl.caseClassTypeDescriptorImpl[T]
```

Comment:
Yes this is there for the TypedTsv, it was in the package for typedtext. The companion object might/should work I think...